docs/deploy.rst

```diff
@@ -265,8 +254,3 @@ the AWS US West datacenter.
     git merge --ff-only origin/master
     git push
     cd ..
-
-* Monitor the push in the ``#mdndev`` IRC channel. The final messages (one
-  for kuma, one for kumascript) are something like::
-
-    🎉 SUCCESS: Check Rollout Status: Branch standby-push build #7
```

Comment:
This could be replaced with:
```
  * Monitor the push in the ``#mdn-infra`` Slack channel. The final messages
    (one for kuma, one for kumascript) look like::

        🎉 SUCCESS Check Rollout Status
        🎉 SUCCESS Record Rollout
```